### PR TITLE
[IA-2660] Bring dependencies to latest

### PR DESCRIPTION
**Pull request by Qi77Qi**
* Since we'll have more development going on in welder....Now is a good time to make sure we're up-to-date with dependencies
* Move to github workflow

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,18 +2,17 @@ import sbt._
...
 object Dependencies {
   val circeVersion = "0.13.0"
-  val http4sVersion = "0.21.7"
-  val grpcCoreVersion = "1.28.0"
-  val scalaTestVersion = "3.2.2"
-  val workbenchGoogle2V = "0.13-39c1b35"
+  val http4sVersion = "0.21.20"
+  val grpcCoreVersion = "1.34.0"
+  val scalaTestVersion = "3.2.7"
+  val workbenchGoogle2V = "0.20-52e271f"
```
Comment (rtitle) [inline project/Dependencies.scala:8]: Latest `google2` will be `0.21-xx` when this is merged btw: https://github.com/broadinstitute/workbench-libs/pull/598

File: project/Settings.scala [hunk 2]
```diff
@@ -40,7 +40,7 @@ object Settings {
 
   // recommended scalac options by https://tpolecat.github.io/2017/04/25/scalac-flags.html
   lazy val commonCompilerSettings = Seq(
-    "-target:jvm-1.8",
+    "-target:jvm-11",
```
Comment (rtitle) [inline project/Settings.scala:43]: Also on L24 you can do `javacOptions ++= Seq("--release", "11"),`

(I think `--release` replaces `-source` and `-target` in Java 11.)

**Review by rtitle — DISMISSED** [review]
Couple comments, looks great

**Comment by Qi77Qi** [comment]
@rtitle addressed all comments...lmk if you have more comments

**Comment by rtitle** [comment]
:shipit:

**Comment by Qi77Qi** [comment]
think I saw this was approved at some point...not sure why it's not showing here...going to merge this assuming @rtitle you're approving with the :shipit: comment

**Pull request merged**